dist/tools/openocd/openocd.sh

```diff
@@ -265,8 +269,8 @@ do_flash() {
     # In case of binary file, IMAGE_OFFSET should include the flash base address
     # This allows flashing normal binary files without env configuration
     if _is_binfile "${IMAGE_FILE}" "${IMAGE_TYPE}"; then
-        # hardwritten to use the first bank
-        FLASH_ADDR=$(_flash_address 1)
+        # Default to using first bank
```

Comment:
I think this comment could be removed since now the value is set via the FLASH_BANK variable.